onnxruntime/test/perftest/performance_runner.cc

```diff
@@ -70,9 +70,10 @@ Status PerformanceRunner::Run() {
   // TODO: end profiling
   // if (!performance_test_config_.run_config.profile_file.empty()) session_object->EndProfiling();
 
-  std::cout << "Total time cost:" << performance_result_.total_time_cost << std::endl
-            << "Total iterations:" << performance_result_.time_costs.size() << std::endl
-            << "Average time cost:" << performance_result_.total_time_cost / performance_result_.time_costs.size() * 1000 << " ms" << std::endl;
+  if (performance_test_config_.run_config.f_verbose)
```

Comment:
We would want to print the timing info even if verbose is disabled, right?

Reply:
A large amount of timing information is printed, this is just extra summary information.  So I marked it verbose.

Reply:
If I just want the summary info, I'll need to turn on verbose that'll give me a lot more than what I want. I guess this one gave me the summary stats without the other detailed timings. Either way works.

Reply:
I put it back.